modules/samples/artifacts/MongoSample/MongoSample.siddhi

```diff
@@ -0,0 +1,96 @@
+@App:name("MongoSample")
```

Comment:
Can we change app name to store-mongodb.siddhi and change the folder name to MongoDBSample